ui/components/component-library/avatar-favicon/avatar-favicon.js

```diff
@@ -65,7 +65,7 @@ AvatarFavicon.propTypes = {
   /**
    * Props for the fallback icon. All Icon props can be used
    */
-  fallbackIconProps: PropTypes.shape(Icon.PropTypes),
+  fallbackIconProps: PropTypes.object,
```

Comment:
Not a huge fan of this and the other options I thought of was:
- Import the deprecated `Icon` component as `IconDeprecated` and use it's propTypes but I'm conscious of those propTypes becoming out of date. 
We can't use the `IconProps` interface until this component has been migrated to TS. So maybe we just go through a short period of unsafe types. I don't think this prop will be regularly used it's more of an escape hatch.

Do you have any suggestions @garrettbear @brad-decker?